data/seeds/04-event_participants-seed.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable func-names */
+
```

Comment:
If I were a more cantankerous man, I'd be annoyed with the excess white-space. But I'm not so I'm just going to approve of it 😂